afs/GettingStarted.md

```diff
@@ -0,0 +1,329 @@
+Project myFirstProject = myFirstFs.getRootFolder()
+         .createFolder("my-first-folder")
+         .createProject("my-first-project");
+```
+
+Everything that we have just created will be persisted to your mapdb file. Your file system tree now looks like:
+```
+my-first-fs
+  +-- my-first-folder
+    +--my-first-project
+```
+
+### Using AFS from groovy scripts
+
+Your configured AFS is also accessible from groovy. This comes in 2 flavours, either with an interactive console using the powsybl shell `powsyblsh`:
+```bash
+powsyblsh
+groovy:000> :register com.powsybl.scripting.groovy.InitPowsybl
+groovy:000> :init_powsybl
+groovy:000> import com.powsybl.contingency.*
+groovy:000>
+```
+
+or using the `itools` command to execute a groovy script:
+```bash
+itools run-script --file my_script.groovy
+```
+
+From groovy code, powsybl provides a variable called `afs` which exposes base methods to access configured file systems. You can then simply perform the same thing as in the java section this way:
+```groovy
+//Create a new folder at the root of you file system, and a new project in that folder.
+myFirstProject = afs.getRootFolder("my-first-fs")
+                    .createFolder("my-first-folder")
+                    .createProject("my-first-project");
+```
+
+
+### Using actual business objects
+
+All this is fine, but the primary goal of AFS is to manage your **business objects**, which we have not seen so far.
+
+AFS is fully extendable with your own type of files, but it already comes with a few basic types for grid studies. The most basic one may be the `ImportedCase` type, which expose a `Network` object to the API.
+
+Such files may only be created inside a project. Projects may be seen as a kind of workspace for a particular study or computation. Inside of a project, we can import a case from a file representing a network, for example an XIIDM file or a UCTE file.
+
+In java:
+```java
+ImportedCase myImportedCase = myFirstProject.getRootFolder()
+              .fileBuilder(ImportedCaseBuilder.class)
+              .withName("my-imported-case")
+              .withFile(Paths.get("path/to/network.xiidm"))
+              .build();
+```
+
+or in groovy with a nice, simplified syntax:
+```groovy
+myImportedCase = myFirstProject.getRootFolder().buildImportedCase {
+              name "my-imported-case"
+              file Paths.get("path/to/network.xiidm")
+            }
+```
+
+Now our tree looks like:
+```
+my-first-fs
+  +-- my-first-folder
+    +-- my-first-project
+      +-- "my-imported-case"
+```
+
+You can then use the methods exposed by your imported case to carry out some business related logic:
+
+```java
+//use stored network
+Network network = myImportedCase.getNetwork();
+//Carry out some computations ...
+...
+//Query stored network to get all substations names
+System.out.println(myImportedCase.queryNetwork(ScriptType.GROOVY,
+                            "network.substationStream.map {it.name} collect()"))
```

Comment:
Are you sure of the syntax here?

Reply:
Yes it works fine! (groovy closure syntax differs from java lambdas if that's what looks weird :) )